addons/common/functions/fnc_loadPerson.sqf

```diff
@@ -29,7 +29,15 @@ if (isNull _vehicle) then {
 };
 
 if (!isNull _vehicle) then {
-    [_unit, true, GROUP_SWITCH_ID, side group _caller] call FUNC(switchToGroupSide);
+    switch (true) do {
+        case ((crew _vehicle isEqualTo []) && {side _caller != side _unit}): {
+            [_unit, true, GROUP_SWITCH_ID, side _caller] call FUNC(switchToGroupSide);
+        };
+        case (side _vehicle != side _unit): {
+            [_unit, true, GROUP_SWITCH_ID, side _vehicle] call FUNC(switchToGroupSide);
+        };
+    };
```

Comment:
`side OBJECT` will report stuff like sideEnemy and sideFriendly. `side GROUP` will only report west, east, resistance, civilian. Are you sure you don't want to keep `side group _something`?

Reply:
Oh yes, my bad. I'll fix it.

Reply:
So will it be fixed and merged? We playing with this "fixed fix" for more than a year. But it is pain in the ass to track our temporary fixes on every ACE release. Would be nice to see solved issue in next release.